modules/smtp/site.js

```diff
@@ -382,14 +382,38 @@ var is_valid_recipient = function(recipient) {
     return recipient.match(valid_regex);
 };
 
-$(function() {    
+var show_cypht_reminder_dialog = function () {
```

Comment:
@Shadow243 can you just use confirm method instead of creating a modal here ? Confirm is used elsewhere in Cypht.

Reply:
@josaphatim  confirm usage added.